### ISSUE TITLE
OAuth login options shouldn't show if they are not properly configured

### ISSUE THREAD

**Issue by KrzysztofMadejski** [consul/consul#2744]
### User Story

# As an admin I want users to be able to login into platform.

### Why is this Needed?

If the oauth logins like twitter or FB are not configured (see below) they are still shown on the login screen and users may try to login with it which will result in an error.

### Description

You may call it the feature to show that these are the possibilites. If so then the documentation https://consul_docs.gitbooks.io/docs/content/en/features/oauth.html should inform how to disable showing them.

### Current Behavior
Not configured oauth
```
  twitter_key: ""
  twitter_secret: ""
  facebook_key: ""
  facebook_secret: ""
  google_oauth2_key: ""
  google_oauth2_secret: ""
```
Social media logins appear on http://consul.kodujdlapolski.pl/users/sign_up.

### Expected Behavior
Social media logins shouldn't appear on http://consul.kodujdlapolski.pl/users/sign_up

### Steps to reproduce
It's a clear docker deployment.

**Comment by markmaas** [comment]
Agreed, without beeing able to remove it, one is basically forced to enable it, otherwise the website is just "broken".

**Comment by javierm** [comment]
@markmaas You can disable these features in the admin section -> Settings -> Global settings -> Features. Here's a link to [enable/disable those features] in our demo.

**Comment by lucialuzuriaga** [comment]
Closing due to inactivity.
